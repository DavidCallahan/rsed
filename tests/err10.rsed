input shell("./catx test1.in")
copy all
