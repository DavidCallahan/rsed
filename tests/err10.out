sh: ./catx: No such file or directory
error in command: ./catx test1.in
